lib/FileWatcher.js

```diff
@@ -1,6 +1,11 @@
 const chokidar = require('chokidar')
 const logger = require('./logger')
 
+Array.prototype.asyncForEach = async function(callback) { // eslint-disable-line
```

Comment:
Ouch, why?

`p-map-series` is probably better fit.

Reply:
Hmm... I missed that. Yeah, probably don't want to modify the prototype.

Reply:
okay thanks @olstenlarck